jpeg/PSPBUILD

```diff
@@ -1,6 +1,6 @@
 # Maintainer: Wouter Wijsman <wwijsman@live.nl>
 pkgname=jpeg
-pkgver=8c
+pkgver=2.1.1
```

Comment:
This change requires ``epoch=1`` to be added to the PSPBUILD, otherwise psp-pacman cannot upgrade from the previous version.

Reply:
Using epoch is not possible if we want to build with the github ci system. It does not support uploading files with ``:`` in the name as artifacts. Makes some sense, since Windows does not support that.